frontend/src/components/CommentsSection/CommentsSection.jsx

```diff
@@ -0,0 +1,46 @@
+import { useQuery, useMutation } from 'react-query';
+import { Section } from 'components/CommentsSection/CommentsSection.styled';
+import { CommentForm } from '../CommentForm/CommentForm';
+import { CommentList } from '../CommentList/CommentList';
+
+export const CommentsSection = () => {
+  const {
+    data: dataQuery,
+    isLoading,
+    isError,
+  } = useQuery('commentList', async () => {
+    return await fetch('/announcements/1/comments').then((response) => response.json());
```

Comment:
Tutaj cyfra powinna zmieniać się dynamiczne, zgadza się?
```suggestion
    return await fetch('/announcements/${announcementId}/comments').then((response) => response.json());
```

Reply:
Zgadza się. Dla testu tak zostało.